Tehtris/tehtris-edr/tests/test_antivirus_alert.json

```diff
@@ -81,10 +81,6 @@
             "architecture": "x86_64"
           },
           "is_server": false
-        },
-        "antivirus": {
-          "total": 65,
-          "positives": 1
```

Comment:
I'm not sure about this modification. Is it intended?

Reply:
It is intended, the deleted lines were a duplicate of lines 64-67